apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPager.swift

```diff
@@ -74,20 +98,50 @@ public class GraphQLQueryPager<InitialQuery: GraphQLQuery, PaginatedQuery: Graph
   }
 
   public var canLoadNext: Bool { canLoadNextSubject.value }
+  public var canLoadPrevious: Bool { canLoadPreviousSubject.value }
 
   public func cancel() {
     Task {
       await pager.cancel()
     }
   }
 
+  public func loadPrevious(
+    cachePolicy: CachePolicy = .fetchIgnoringCacheData,
+    completion: (@MainActor (Error?) -> Void)? = nil
+  ) {
+    Task<_, Never> {
+      do {
+        try await pager.loadPrevious(cachePolicy: cachePolicy)
+        await completion?(nil)
+      } catch {
+        await completion?(error)
+      }
+    }
+  }
+
   public func loadMore(
```

Comment:
I like the function being called `loadMore()` for the single direction case. But since with bidirectional pagination everything else is labeled as `previous` and `next`, I wonder if we should rename this to `loadNext()`.

There is also the option of adding a `loadNext()` function that just calls `loadMore()`.